spikeinterface/comparison/collisioncomparison.py

```diff
@@ -118,11 +118,19 @@ def compute_collision_by_similarity(self, similarity_matrix, unit_ids=None, good
         recall_scores = []
         similarities = []
         pair_names = []
+
+        performances = self.get_performance()['accuracy']
+
         for r in range(n):
             for c in range(r + 1, n):
 
                 u1 = unit_ids[r]
                 u2 = unit_ids[c]
+
+                if good_only:
+                    if (performances[u1] < 0.9) or (performances[u2] < 0.9):
```

Comment:
This 0.9 should be variable exposed in the function.

Reply:
I'll remove this modification  for now, but this is something that I'll leave commented, because it might be less strict that the current removal we were doing for the paper